src/flyte/_bin/runtime.py

```diff
@@ -142,8 +142,9 @@ def main(
     bundle = None
     if tgz or pkl:
         bundle = CodeBundle(tgz=tgz, pkl=pkl, destination=dest, computed_version=version)
-    init(org=org, project=project, domain=domain, image_builder="remote", **controller_kwargs)
```

Comment:
We created a gRPC channel twice. I think we should be able to reuse it in the controller.